api/services/gameService.js

```diff
@@ -175,6 +177,9 @@ module.exports = {
         gameUpdates.result = GameResult.P1_WINS;
       }
       await Game.updateOne({ id: game.id }).set(gameUpdates);
+      if (game.ranked) {
+        await sails.helpers.addGameToMatch(game);
+      }
```

Comment:
Fast fail makes more sense here imo:
```suggestion
      if (!game.ranked) {
          return res;
      }
      await sails.helpers.addGameToMatch(game);
```

Reply:
Interesting tradeoff I'd never really considered, but early return adds a line (because the same return statement is duplicated), but the late-return involves another level of indentation for the bulk of the function body. Low FIG score for me, but I'll take your suggestion and return early

Reply:
Wait actually I take that back. Because there's only one line inside the conditional block either way, the first way is the same indentation for the same blocks of code, but is one line shorter. Why return early?

Reply:
I think that's reasonable, but because the entire rest of the function is just closing braces and the same return statement, I'm gonna keep it this way atm